Abbus July Adjust Adventure

AbbusF opened the pull request:
-New 3 doctrine adjusted with the feedback you guys gave
-30000 infantry equipment is added to Soviet Union
-Soviet Union AI will avoid naval invading Norway now, I hope

AbbusF committed:
Update load_13.dds

AbbusF committed:
Update infantry.txt

AbbusF committed:
New doc

AbbusF committed:
a

AbbusF committed:
Update r56e_land_doctrine_l_english.yml

AbbusF committed:
b

AbbusF committed:
Update land_doctrine.txt

AbbusF committed:
Merge branch 'Beta-Edits' into abbus_gfx_july

AbbusF committed:
Add starting equipment for soviet onion

AbbusF committed:
Merge branch 'Beta-Edits' into abbus_gfx_july

AbbusF committed:
Update land_doctrine.txt

AbbusF committed:
tonk

AbbusF committed:
oof

AbbusF committed:
load

AbbusF committed:
Merge branch 'Beta-Edits' into abbus_gfx_july

AbbusF committed:
Update CHI.txt

AbbusF committed:
Break AI chain

AbbusF committed:
1

AbbusF committed:
AI revamped

AbbusF committed:
1

AbbusF committed:
Update land_doctrine.txt

AbbusF committed:
Merge branch 'Beta-Edits' into abbus_gfx_july

AbbusF committed:
Merge branch 'abbus_gfx_july' of https://github.com/deliciousmods/1956_beta into abbus_gfx_july

AbbusF committed:
Update land_doctrine.txt

TheBrit7 reviewed (changes requested):
This branch has conflicts, please fix.

AbbusF committed:
Merge branch 'Beta-Edits' into abbus_gfx_july

AbbusF committed:
Update Portrait_Vietnam_Van_Tien_Dung.dds

AbbusF committed:
Merge branch 'abbus_gfx_july' of https://github.com/deliciousmods/1956_beta into abbus_gfx_july

AbbusF committed:
Buff

-Militia heavy buff
-Double envelopment minor buff
-Infantry Anti Tank buff
-Flamethrower buff
-Sniper Rifle buff
-Clothing buff
-Camo buff

AbbusF committed:
Update land_doctrine.txt

AbbusF committed:
5 new tactics

AbbusF committed:
Merge branch 'Beta-Edits' into abbus_gfx_july

AbbusF committed:
Update electronic_mechanical_engineering.txt

AbbusF committed:
Merge branch 'Beta-Edits' into abbus_gfx_july

AbbusF committed:
Merge branch 'Beta-Edits' into abbus_gfx_july

AbbusF committed:
Merge branch 'Beta-Edits' into abbus_gfx_july

AbbusF committed:
Merge branch 'abbus_gfx_july' of https://github.com/deliciousmods/1956_beta into abbus_gfx_july
